storage_controller/src/main.rs

```diff
@@ -216,8 +216,8 @@ struct Cli {
     /// Number of safekeepers to choose for a timeline when creating it.
     /// Safekeepers will be choosen from different availability zones.
     /// This option exists primarily for testing purposes.
-    #[arg(long, default_value = "3", value_parser = clap::value_parser!(i64).range(1..))]
-    timeline_safekeeper_count: i64,
+    #[arg(long, default_value = "3", value_parser = clap::value_parser!(u64).range(1..))]
+    timeline_safekeeper_count: usize,
```

Comment:
why is there an inconsistency here between the value parser and the type?

Reply:
Setting u64 doesn't work in runtime (it panics). Setting `RangedU64ValueParser` manually works, though.
But it now looks not so pretty, I'm not sure if we need it at all...